app/assess/routes.py

```diff
@@ -372,8 +376,8 @@ def application(application_id):
             abort(404)
         assessor_task_list_metadata["fund_name"] = fund.name
         state = AssessorTaskList.from_json(assessor_task_list_metadata)
-
-    determine_display_status(state, flag)
+    is_flaggable = not flag or flag.flag_type == FlagType.RESOLVED
```

Comment:
Might this be
 flag.flag_type in (FlagType.RESOLVED, FlagType.QA_COMPLETE)
as we can still flag qa complete applications

Reply:
Good spot, fixed in 85ee0e9